testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/util/MailUtils.java

```diff
@@ -47,7 +47,13 @@ public static String getPasswordResetEmailLink(MimeMessage message) throws IOExc
 
     public static String getPasswordResetEmailLink(EmailBody body) throws IOException, MessagingException {
         final String textChangePwdUrl = getLink(body.getText());
-        final String htmlChangePwdUrl = getLink(body.getHtml());
+        String htmlChangePwdUrl = getLink(body.getHtml());
```

Comment:
Why is this needed? The link shouldn't be sanitized right?

Reply:
I don't have a problem with it being sanitized, but if it's not a valid link afterwards that doesn't seem correct

Reply:
It is a valid link.  I tried it to make sure.